akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/ProtoAnySerialization.scala

```diff
@@ -451,3 +466,46 @@ import scalapb.options.Scalapb
   }
 
 }
+
+/**
+ * Primarily intended for replication where there is no public protocol
+ *
+ * INTERNAL API
+ */
+@InternalApi
+private[akka] final class DelegateToAkkaSerialization(system: ActorSystem[_]) extends AkkaProjectionGrpcSerialization {
+  import ProtoAnySerialization._
+  private val serialization = SerializationExtension(system.classicSystem)
+
+  override def serialize(event: Any): ScalaPbAny = {
+    val anyRefEvent = event.asInstanceOf[AnyRef]
+    val serializer = serialization.findSerializerFor(anyRefEvent)
+    val bytes = serializer.toBinary(anyRefEvent)
+    val manifest = Serializers.manifestFor(serializer, anyRefEvent)
+    val id = serializer.identifier
+    ScalaPbAny(akkaSerializationTypeUrl(id, manifest), UnsafeByteOperations.unsafeWrap(bytes))
+  }
+
+  override def deserialize(event: ScalaPbAny): Any = {
```

Comment:
I haven't looked in detail at this PR, but could we keep deserialization side as it was? If we serialize with AkkaSerializationTypeUrlPrefix that would still be deserialized correctly?

Reply:
I think that if we can have no special casing that is better and less trickery to try to understand, all messages are fed to the configured akka serializer. The any-passthrough is such a weird special case that it is very unlikely that anybody else than us ourselves is doing that.

Reply:
ok